src/components/templates/FireBarrel/FireBarrel.tsx

```diff
@@ -24,9 +24,11 @@ export const FireBarrel: React.FC = () => {
     venueId: venue?.id,
   });
 
+  const recentUserCount = venue?.recentUserCount ?? 0;
+
   const seatCount =
-    recentVenueUsers?.length > DEFAULT_BURN_BARREL_SEATS
-      ? recentVenueUsers.length
+    recentUserCount > DEFAULT_BURN_BARREL_SEATS
```

Comment:
Is `DEFAULT_BURN_BARREL_SEATS` used in staging?

Reply:
It seems to only be used in `Firebarrel`.
![image](https://user-images.githubusercontent.com/63194656/133225275-23d2f15d-0a79-4f88-b61b-1baf61c1c100.png)

It looks like legacy logic, but probably some people depend on its logic, so I don't want to remove it